src/core/client/auth/components/ForgotPassword.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { StatelessComponent } from "react";
+import { Field, Form } from "react-final-form";
+import { OnSubmit } from "talk-framework/lib/form";
+import * as styles from "./SignIn.css";
+
+import {
+  composeValidators,
+  required,
+  validateEmail,
+} from "talk-framework/lib/validation";
+
+import {
+  Button,
+  Flex,
+  FormField,
+  InputLabel,
+  TextField,
+  Typography,
+  ValidationMessage,
+} from "talk-ui/components";
+
+interface FormProps {
+  email: string;
+}
+
+export interface ForgotPasswordForm {
+  onSubmit: OnSubmit<FormProps>;
+}
+
+const ForgotPassword: StatelessComponent<ForgotPasswordForm> = props => {
+  return (
+    <Form onSubmit={props.onSubmit}>
+      {({ handleSubmit }) => (
+        <form autoComplete="off" onSubmit={handleSubmit}>
+          <Flex itemGutter direction="column" className={styles.root}>
```

Comment:
This should be `itemGutter="double"`, once we fixed the margins and paddings of the `FormField` component (it has an undesired margin-top) by using the `Flex` component (see other comment in the `FormField.ts`).

Reply:
This will also make the `<div className={styles.footer}>` obsolete.

Reply:
👍